gitbot.rb

```diff
@@ -30,6 +30,29 @@ def pr_test(upstream, pr_sha_com, repo, pr_branch)
   git.del_pr_branch(upstream, pr_branch)
 end
 
+def check_if_changes_files_changed(repo, pr)
+  if @changelog_test
```

Comment:
we should check here if the .change.md/txt got modified by the pr, and return success if it is the case.

- i would add on the check that we strip all the whitespaces because these are not changes ( it happens when maybe you open with vim and do no modification or other utilisat. errors)

in this case, i would add like


```
@pr_files.each do |pr_file|
   if pr_file == '.changes.md'
     # open the file
     # check that the changes are not only whitespaces 
   end
```

Reply:
I tested it together with using option
```
  -f .changes
```
But I agree that we should just enforce it when we use --changelogtest.

Reply:
ok we can open an issue, for tracking this and do it later

Reply:
No, you just create too many issues :-). If you want it, I can just change it.

Reply:
:) i think that atm  this is ok, even to have with whitespaces. to have it without imply an extra call from commits from octokit, for me is not 99 % needed at this stade. (that why the issue :)  )